storage/lrucache/lrucache.go

```diff
@@ -65,7 +65,7 @@ func (c *LRUCache) RegisterHandler(handler func(key []byte)) {
 // Get looks up a key's value from the cache.
 func (c *LRUCache) Get(key []byte) (value interface{}, ok bool) {
 	v, ok := c.cache.Get(string(key))
-	if ok == false {
+	if !ok {
```

Comment:
maybe you can just _return c.cache.Get(string(key))_

Reply:
done